source/_data/plugins.yml

```diff
@@ -2723,10 +2723,18 @@
     - card
     - crawl
     - spider
+- name: hexo-highcharts
+  description: "A hexo tag for inserting highcharts chart in your blog."
+  link: https://github.com/lilunze/hexo-highcharts
+  tags:
+    - tag
+    - highcharts
+    - chart
 - name: hexo-image-cloudflare
   description: Use Cloudflare to cache images in Hexo
   link: https://github.com/njzjz/hexo-image-cloudflare
   tags:
     - image
     - cloudflare
     - cdn
+
```

Comment:
Would you mind removing the extra blank line?

Reply:
Can be removed

Reply:
You might add two blank lines, we just hope  you to remove one of these lines 😂